sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -247,6 +248,28 @@ struct {
 ```
 </td>
 </tr>
+<tr>
+<td>image</td>
+<td>
+
+``` C++
+ze_image_handle_t
+```
+</td>
+<td>
+
+``` C++
+struct {
+    ze_image_handle_t ZeImageHandle;
+    sycl::image_channel_order ChanOrder;
+    sycl::image_channel_type ChanType;
+    range<Dimensions> Range;

```

Comment:
```suggestion
    sycl::range<Dimensions> Range;
```
And how is `Dimensions` defined?

Reply:
The discussion of `make_image` ( starting at line 460 ) has a more full description, explaining why this struct has the members that it has, and what is expected. It felt like that was a more appropriate place for the discussion and I didn't want to duplicate it.  I notice the other interop handles document themselves similarly - with spare descriptions in this table, and more explication lower.

Reply:
at least adding `sycl::` will not harm, I'll let @gmlueck suggest how we better describe the templated input

Reply:
Yes, I added the sycl:: to range.

Reply:
> at least adding sycl:: will not harm, I'll let @gmlueck suggest how we better describe the templated input

@smaslov-intel: Can you elaborate on what should be better described?

Reply:
@gmlueck : the "Dimensions" template argument:

> sycl::range<**Dimensions**> Range;

Reply:
The description of `make_image` says this:

> where the Range should be ordered (width), (width, height), or (width, height, depth) for 1D, 2D and 3D images respectively, with those values matching the dimensions used in the `ze_image_desc` that was used to create the `ze_image_handle_t` initially

Doesn't that adequately describe the `Dimensions`?